pdr_backend/trader/trader_agent.py

```diff
@@ -42,11 +42,14 @@ def __init__(
         self.cache = Cache(cache_dir=cache_dir)
         self.load_cached_epochs()
 
+        print("\n" + "-" * 80)
+        print("Config:")
+        print(self.config)
+
```

Comment:
Also list all the feeds at the beginning, with full detail. See example in predictoor/approach1/predictoor_agent1.py